package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "urban-airship-cli",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
You only pump version (using `npm run utils deploy 1.1.0`) after your pull request is merged. But as you already did it just keep this in your mind @arthursz